core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackSyncService.java

```diff
@@ -206,24 +206,20 @@ private void deleteResourceIfNeeded(Stack stack, InstanceMetaData instance) {
     }
 
     private void handleSyncResult(Stack stack, Map<InstanceSyncState, Integer> instanceStateCounts, boolean stackStatusUpdateEnabled) {
-        Set<InstanceMetaData> instances = instanceMetaDataRepository.findNotTerminatedForStack(stack.getId());
         if (instanceStateCounts.get(InstanceSyncState.UNKNOWN) > 0) {
             eventService.fireCloudbreakEvent(stack.getId(), AVAILABLE.name(),
                     cloudbreakMessagesService.getMessage(Msg.STACK_SYNC_INSTANCE_STATUS_COULDNT_DETERMINE.code()));
         } else if (instanceStateCounts.get(InstanceSyncState.IN_PROGRESS) > 0) {
             eventService.fireCloudbreakEvent(stack.getId(), AVAILABLE.name(),
                     cloudbreakMessagesService.getMessage(Msg.STACK_SYNC_INSTANCE_OPERATION_IN_PROGRESS.code()));
-        } else if (instanceStateCounts.get(InstanceSyncState.RUNNING) > 0 && instanceStateCounts.get(InstanceSyncState.STOPPED) > 0) {
+        } else if ((instanceStateCounts.get(InstanceSyncState.RUNNING) >= 0 || (instanceStateCounts.get(InstanceSyncState.DELETED_ON_PROVIDER_SIDE) >= 0))
```

Comment:
I thought we agreed to put the status to STOPPED_FAILED if we find deleted instances. https://jira.cloudera.com/browse/ENGESC-3354?focusedCommentId=3950763&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3950763
Has this changed?

Reply:
No, we agreed it remain STOPPED